app/generate_csv_eventyay.py

```diff
@@ -14,11 +14,13 @@
 
 def tocsv(url_eventyay, filename):
     with urlopen(url_eventyay) as url:
-        data = json.loads(url.read().decode(url.info().get_param('charset') or 'utf8'))
+        data = json.loads(url.read().decode(
+            url.info().get_param('charset') or 'utf8'))
```

Comment:
Don't you think this is just messing up the files and making it difficult to read?

Reply:
@jajodiaraghav I will do it by tomorrow. 😄